azext_iot/common/basic_sas_token_auth.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+basic_sas_token_auth: Module containing Basic Shared Access Signature token class.
+
+"""
+from msrest.authentication import Authentication
+
+
+class BasicSasTokenAuthentication(Authentication):
```

Comment:
Rather than make a new module for this variant of auth, please put and reference this class in the existing `sas_token_auth.py` module.